src/Generator/JsonGenerator.php

```diff
@@ -89,6 +89,8 @@ private function generateToc(MetaEntry $metaEntry, ?array $titles): array
         foreach ($titles as $title) {
             $tocTree[] = [
                 'url' => sprintf('%s#%s', $metaEntry->getUrl(), Environment::slugify($title[0])),
+                'page' => u($metaEntry->getUrl())->beforeLast('.html'),
+                'fragment' => Environment::slugify($title[0]),
```

Comment:
Bah, apparently we never created a test for this class. Ideally, we *would* have a test for this... probably most useful would be an integration test. But, the feature seems good to me 👍